build_component.py

```diff
@@ -16,17 +16,16 @@
     to a specific branch, pull from remotes, etc.
     (In a future version of this script, the currently active branch 
     for each repo will be displayed as a visual reminder of this fact.)
-
+    
 <Usage>
   build.py  [-t] [-v] [-c] [-r] <target_directory>
```

Comment:
We should explain that `build_component.py` is normally indirectly executed by a script like `build.py`.